wlct/tournaments.py

```diff
@@ -2448,6 +2455,9 @@ def finish_game(self):
             self.game_finished_time = timezone.now()
             self.save()
 
+        else:
+            log('Could not find tournament teams with id\'s: {} {} in tournament {}'.format(team1, team2, self.tournament.id), LogLevel.error)
```

Comment:
please use log_tournament, which takes the actual tournament in as an arg too for better logging.

Reply:
Done